sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec

```diff
@@ -8,7 +8,7 @@
     <owners>SonarSource</owners>
     <license type="file">License.txt</license>
     <projectUrl>http://vs.sonarlint.org</projectUrl>
-    <iconUrl>http://dist.sonarsource.com/csharp/download/ico-sonarlint-64.png</iconUrl>
+    <icon>https://sonarsource.github.io/analyzers-github-static-resources/sonar-dotnet/SonarLint-icon-64px.png</icon>
```

Comment:
@andrei-epure-sonarsource In order to use the `<icon>` element, we need to have at least version 5.3 of NuGet (See https://github.com/NuGet/Home/issues/8660).
I suggest we revert the update of `<iconUrl>` to `<icon>` and create a ticket for updating NuGet version and `iconUrl` element. WDYT ?

Reply:
@andrei-epure-sonarsource As discussed and agrred on, we will keep `<iconUrl>` element until we update NuGet version on our build pipeline. I will update the PR accordingly

Reply:
thanks! 👍